styles.css

```diff
@@ -0,0 +1,88 @@
+:root {
+  --white-color: white;
+}
+
+.body {
+  background-color: #1c1a19;
+  margin: 0;
+}
+
+.header {
+  padding: 0.5em;
+  background-color: #3c3a39;
+  position: fixed;
+  width: 99%;
+  height: 92px;
+}
+
+.container {
+  display: flex;
+  place-content: space-between;
+}
+
+.name {
+  color: var(--white-color);
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-size: 18px;
+  line-height: 24px;
+  width: 77px;
+  height: 24px;
+  padding: 1em;
+}
```

Comment:
- Your navbar still doesn't look like the Figma template, this is partly because you have given your logo/name a limited with. Please try to fix this. 🙏 🙏 

Your project looks like this;
![nav current](https://user-images.githubusercontent.com/73607512/191811135-398c1c18-c039-4e38-bdd6-a5796c88a0aa.PNG)